script.js

```diff
@@ -121,4 +123,33 @@ form.addEventListener('submit', (event) => {
     event.preventDefault();
     error.innerText = 'Please Your Email Must be in Lower Case';
   }
-});
\ No newline at end of file
+});
+// Populate LocalStrorage
+function populateStorage() {
+  formObj = {
+    Name: form.fullname.value,
+    Email: form.email.value,
+    Message: form.message.value,
+  };
+  const convertObj = JSON.stringify(formObj);
+  localStorage.setItem('FormData', convertObj);
+}
+
+// Populate FormFields
+function getDataFromLocalStorage() {
+  let data = null;
+  data = localStorage.getItem('FormData');
+  data = JSON.parse(data);
+  form.fullname.value = data.Name;
+  form.email.value = data.Email;
+  form.message.value = data.Message;
+}
+
+window.onload = function () {
+  form.user.addEventListener('input', populateStorage);
+  form.email.addEventListener('input', populateStorage);
+  form.message.addEventListener('input', populateStorage);
+  if (localStorage.getItem('FormData') !== null) {
+    getDataFromLocalStorage();
+  }
```

Comment:
Nice job 👍 here getting the local storage data on page load. Please fix the above issues I mentioned.